test/helpers/kubernetes-upstream.ts

```diff
@@ -73,9 +73,11 @@ export async function validateUpstreamStoredMetadata(
   remainingChecks: number = UPSTREAM_POLLING_CONFIGURATION.MAXIMUM_REQUESTS,
 ): Promise<boolean> {
   while (remainingChecks > 0) {
-    console.log(
-      `Pinging upstream for existing metadata (${remainingChecks} checks remaining)...`,
-    );
+    if (remainingChecks % 10 === 0) {
```

Comment:
What is this check for?

Reply:
this is to stop the logs being spammed up to 180 times each time this function is called - only logging every 10th time